README.rst

```diff
@@ -0,0 +1,31 @@
+CIbyl
+=====
+
+Command line interface for querying CI environments
+
+Installation
+************
+
+```
+cd cibyl 
+virtualenv ~/.cibyl_venv && source ~/.cibyl_venv/bin/activate
+sudo pip install .
```

Comment:
Should we add the "pip install -r requirements.txt" maybe?

Reply:
Isn't it installed when running "sudo pip install ."? not sure

Reply:
Just tried and no.

Reply:
changed to "sudo pip install cibyl". When installing from PyPi, it will install the requirements as well